live-examples/css-examples/text/text-wrap.html

```diff
@@ -0,0 +1,44 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="text-wrap">
+  <div class="example-choice" initial-choice="true">
+    <pre><code class="language-css">text-wrap: wrap;</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Copy to Clipboard</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">text-wrap: nowrap;</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Copy to Clipboard</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">text-wrap: balance;</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Copy to Clipboard</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">text-wrap: pretty;</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Copy to Clipboard</span>
+    </button>
+  </div>
+
+  <div class="example-choice">
+    <pre><code class="language-css">text-wrap: stable;</code></pre>
+    <button type="button" class="copy hidden" aria-hidden="true">
+      <span class="visually-hidden">Copy to Clipboard</span>
+    </button>
+  </div>
+</section>
+
+<div id="output" class="output large hidden">
+  <section id="default-example" class="default-example">
+    <div id="example-element" class="transition-all">
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem aut cum eum id quos est.</p>
```

Comment:
I think it would make sense to add `contenteditable` to the paragraph, and perhaps also add a little note to tell readers to experiment with editing the text. I say this because some values of `text-wrap` (most notably `stable`) only have an effect when editing user input.

Reply:
Hi @chrisdavidmills ! Thanks for the suggestions, I agree with you.
Also, I added a wrapper `<div>` for a better look:
![Image](https://github.com/mdn/interactive-examples/assets/123000937/934a1a7c-1b89-4152-bcd5-30c77a95035a)

Reply:
Look great, thanks @gldaniil.

I do wonder whether "Click here..." might confuse people into thinking they have to click the text above the box rather than the box itself. How about "Edit the text in the box" or similar?

Reply:
Yeah, you're right, I considered different versions of this phrase to make it short and informative, but I think your choice looks more appropriate. Thanks @chrisdavidmills!